src/Utils/ProtectedRoutes.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+
+interface ProtectedRoutesProps {
+  component: React.FunctionComponent;
+  path: string;
+}
+
+export const ProtectedRoutes: React.FC<ProtectedRoutesProps> = (props: ProtectedRoutesProps) => {
+  const bool = true;
+  const { component: Component } = props;
+  return bool ? <Component /> : <Redirect to={{ pathname: "/login" }} />;
```

Comment:
I guess you want to check if users are authenticated and put this in `bool`, right? If so, please add a remark in the authentication issue so that we don't forget that this needs to be changed here.